profiles/apparmor/template.go

```diff
@@ -2,6 +2,12 @@
 
 package apparmor // import "github.com/docker/docker/profiles/apparmor"
 
+// NOTE: This profile is replicated in containerd and libpod. If you make a
+//       change to this profile, please make follow-up PRs to those projects so
+//       that these rules can be synchronised (because any issue with this
+//       profile will likely affect libpod and containerd).
+// TODO: Move this to a common project so we can maintain it in one spot.
```

Comment:
Is it (in its current form) generic enough to be re-used, or are other changes needed?


Perhaps would be good to open a ticket for this for further discussion

Reply:
I'm about to submit a patch to containerd which is missing these changes (in fact it's missing all of the signal changes from 2018!) and I will propose it there.

Reply:
The package itself has received little updates, so from that perspective I don't think it'd be a problem to move it separate (we moved out some other packages, such as https://github.com/moby/sys, for easier reusability)